suites/pacific/cephadm/test_validate_cephadm_bootstrap_cases.yaml

```diff
@@ -95,3 +95,9 @@ tests:
           playbook: playbooks/bootstrap-cluster-overide-exisitng-keys.yml
           module_args:
             mon_node: node1
+
```

Comment:
As we are creating separate test suite file, this test not required now.

Reply:
Noted, will remove it